core/watcher/eventsource/package_test.go

```diff
@@ -129,3 +129,7 @@ func (e changeEvent) Namespace() string {
 func (e changeEvent) Changed() string {
 	return e.changed
 }
+
+func singleton(s string) []string {
```

Comment:
Weird choice of name, could be mixed up with the singleton pattern :rofl:  https://refactoring.guru/design-patterns/singleton